scripts/solr_builder/solr_builder/solr_builder_main.py

```diff
@@ -64,6 +115,7 @@ def __init__(self, db_conf_file):
         self._db_conf = config_section_to_dict(db_conf_file, "postgres")
         self._conn = None  # type: psycopg2._psycopg.connection
```

Comment:
`Optional[psycopg2._psycopg.connection]` ;-)

Reply:
:D Wasn't me!

(Well actually it was me, but it wasn't me _right now_ :P )